### PR TITLE
Add 3 new Halloween jobs

### PR DESCRIPTION
**Pull request by JORJ949**
## About the PR 
I took the ideas I liked from this forum post: https://forum.ss13.co/showthread.php?tid=23136 and made them

image
Discount Wizard:
Starts with bathrobe, slippers, apprentice cap and mop, and also gets unsynchronized biomass manipulation as discount PHEE CABUE

image
Super Spy:
Starts with a suit, eyepatch, sneaking suit, cardboard box and chameleon for hiding with box

image
Space Cowboy:
Starts with a poncho, badge, cigarette and a special belt for keeping your (foam) sidearm in and becomes a cow
NOTE: This already existed as a special job, but I moved it to halloween and made some changes.

## Why's this needed? 
More funny gimmick jobs for spooktober 

## Changelog 

```changelog
(u)JORJ949
(*)Added 3 new jobs to the Spooktober lineup; Discount Wizard, Super Spy, and Space Cowboy.
```

### CODE REVIEW

**Comment by MylieDaniels** [comment]
Not a fan of the HoS exclusive janitor job that gets an item removed for being comically too good. Other ones seem neat.

**Comment by JORJ949** [comment]
Tsunami replaced with WAV3

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -2120,6 +2101,66 @@ ABSTRACT_TYPE(/datum/job/special/halloween)
...
+	slot_suit = list(/obj/item/clothing/suit/bathrobe)
+	slot_head = list(/obj/item/clothing/head/apprentice)
+	slot_foot = list(/obj/item/clothing/shoes/fuzzy)
+	items_in_backpack = list(/obj/item/mop)
+
+	special_setup(var/mob/living/carbon/human/M)
+		..()
+		if (!M)
+			return
+		M.bioHolder.AddEffect("melt", magical=1)
```
Comment (Glamurio) [inline code/datums/jobs.dm:2145]: Concerned about this being magical.

File: code/datums/jobs.dm [hunk 2]
```diff
@@ -2120,6 +2101,66 @@ ABSTRACT_TYPE(/datum/job/special/halloween)
...
+	slot_eyes = list(/obj/item/clothing/glasses/eyepatch)
+	slot_suit = list(/obj/item/clothing/suit/armor/sneaking_suit/costume)
+	slot_foot = list(/obj/item/clothing/shoes/swat)
+	items_in_backpack = list(/obj/item/clothing/suit/cardboard_box )
+
+	special_setup(var/mob/living/carbon/human/M)
+		..()
+		if (!M)
+			return
+		M.bioHolder.AddEffect("chameleon", magical=1)
```
Comment (Glamurio) [inline code/datums/jobs.dm:2162]: Likewise concerned about this being magical.

**Review by Glamurio — CHANGES_REQUESTED** [review]

File: code/datums/jobs.dm (hunk 1 above)
Comment (JORJ949) [reply code/datums/jobs.dm:2145]: the other halloween jobs do it this way

Comment (Glamurio) [reply code/datums/jobs.dm:2145]: While that is true, I think biomass manipulation can be really annoying to deal with. Arguably it's unsynchronized. Maybe it's fine.

File: code/datums/jobs.dm [hunk 3]
```diff
@@ -2120,6 +2101,66 @@ ABSTRACT_TYPE(/datum/job/special/halloween)
 		M.bioHolder.AddEffect("pickle", magical=TRUE)
 		M.blood_id = "juice_pickle"
 
+/datum/job/special/halloween/cowboy
+	name = "Space Cowboy"
+	limit = 1
+	wages = PAY_UNTRAINED
+	receives_badge = TRUE
```
Comment (flrsh) [inline code/datums/jobs.dm:2108]:
```suggestion
	wages = PAY_UNTRAINED
	starting_mutantrace = /datum/mutantrace/cow
	receives_badge = TRUE
```

File: code/datums/jobs.dm [hunk 4]
```diff
@@ -2120,6 +2101,66 @@ ABSTRACT_TYPE(/datum/job/special/halloween)
...
+	slot_card = /obj/item/card/id/civilian
+	slot_poc1 = list(/obj/item/device/pda2/botanist)
+	slot_poc2 = list(/obj/item/device/light/zippo/gold)
+	slot_back = list(/obj/item/storage/backpack/satchel/brown)
+
+	special_setup(var/mob/living/carbon/human/M)
+		..()
+		if (!M)
+			return
+		M.bioHolder.AddEffect("cow", magical=1)
```
Comment (flrsh) [inline code/datums/jobs.dm:2127]:
```suggestion
```

File: code/datums/jobs.dm (hunk 1 above)
Comment (flrsh) [reply code/datums/jobs.dm:2145]: Hm yeah I'm torn. If people are obnoxious with it we can unmagical it maybe?

**Review by flrsh — APPROVED** [review]

**Pull request merged**